torch/fx/experimental/GraphManipulation.py

```diff
@@ -0,0 +1,7 @@
+def replace_all_uses_with(fx_module, old_op, old_target, new_op, new_target):
+    """Modifies all nodes  in nodes which match the op code and target of old_node, and updates them to match new_node"""
```

Comment:
I think the name of this function might be confusing, since "replace all uses with" in TorchScript IR, for example, replaces uses of a value, rather than swapping out node (properties). Maybe something like `swap_by_target`? (not a great name but the best i could think of)